includes/news/v2/mail/featured-stories.php

```diff
@@ -1,49 +1,62 @@
-			<td style="padding-bottom: 30px; padding-top: 30px; padding-left: 0; padding-right: 0;<?php if( $index < $total - 1 && $index < $limit ) echo " border-bottom: solid 1px #aaa;"; ?>">
-				<table class="tableCollapse" width="600" border="0" align="center" style="border-spacing: 0; border-collapse: collapse;">
-					<tbody>
-						<tr>
-							<td style="padding-left: 0; padding-right: 0;">
-								<a href="<?php echo $permalink; ?>">
-									<img class="responsiveimg" border="0" width="600" style="border:none;" src="<?php echo $image; ?>" />
-								</a>
-							</td>
-						</tr>
-						<tr>
-							<td class="montserratbold" style="padding-left: 0; padding-right: 0; font-family: Helvetica, Arial, sans-serif; font-size: 24px; font-weight: bold; padding-top: 15px; padding-bottom: 4px; line-height: 1.1; color: #000; text-align: left;" align="left">
-							<a href="<?php echo $permalink; ?>" style="color: #000; text-decoration: none;"><?php echo $title; ?></a>
-							</td>
-						</tr>
-						<tr>
-							<td class="montserratlight" style="padding-left: 0; padding-right: 0; font-family: Helvetica, Arial, sans-serif; font-size: 16px; line-height: 1.4; color: #000; text-align: left;" align="left">
-							<?php echo $description; ?>
-							<a href="<?php echo $permalink; ?>" style="color: #006699; text-decoration: underline;">Read More</a>
-							</td>
-						</tr>
-					</tbody>
-				</table>
-			</td>
-		</tr>
-		<?
+		if( $index == $limit || $index == $total ) break;
+		if( $index > 0 && $index % 2 == 0 ) echo "</tr><tr>";
+		?>
+		<th class="columnCollapse" align="left" width="290" style="font-family: Helvetica, Arial, sans-serif; color: #848484; padding-left: 10px; padding-right: 10px; padding-top: 30px; padding-bottom: 0; vertical-align: top; text-align: center;">
+			<table class="tableCollapse" width="100%" style="width: 100%; border-spacing: 0; border-collapse: collapse;"><tbody>
+				<tr>
+					<td>
+						<table width="100%" border="0" align="center" style="border-spacing: 0; border-collapse: collapse;">
+							<tbody>
+								<tr>
+									<td style="padding-left: 0; padding-right: 0;">
+										<a href="<?php echo $permalink; ?>">
+											<img class="responsiveimg" border="0" width="290" style="border:none;" src="<?php echo $image; ?>" />
+										</a>
+									</td>
+								</tr>
+								<tr>
+									<td class="montserratbold" style="padding-left: 0; padding-right: 0; font-family: Helvetica, Arial, sans-serif; font-size: 24px; font-weight: bold; padding-top: 15px; padding-bottom: 4px; line-height: 1.1; color: #000; text-align: left;" align="left">
+										<a href="<?php echo $permalink; ?>" style="color: #000; text-decoration: none;"><?php echo $title; ?></a>
+									</td>
+								</tr>
+								<tr>
+									<td class="montserratlight" style="padding-left: 0; padding-right: 0; font-family: Helvetica, Arial, sans-serif; font-size: 16px; line-height: 1.4; color: #000; text-align: left;" align="left">
```

Comment:
This `td`'s contents end up inheriting the 700 font weight from the parent `th`--you'll probably need to set an explicit `font-weight: 400` here to make sure the weight displays as expected across email clients and browsers.  Looking in Firefox I see the additional bolding take effect; Chrome is _slightly_ bolder but almost not noticeable.